tests/docker/build.sh

```diff
@@ -19,7 +19,7 @@ set -e
 
 # build env
 apt update
-DEBIAN_FRONTEND="noninteractive" apt install -y build-essential libjpeg-dev libpng-dev libssl-dev ninja-build cmake pkg-config git perl vim meson cargo cargo-c nasm
+DEBIAN_FRONTEND="noninteractive" apt install -y build-essential cargo cargo-c cmake git libjpeg-dev libpng-dev libssl-dev libyuv-dev meson nasm ninja-build perl pkg-config vim
```

Comment:
The cmake command at line 74 should be updated to say `-DAVIF_CODEC_AOM=SYSTEM`, etc. We can do that in a separate pull request.

Reply:
Indeed. BTW, what is the point of that file: shouldn't we compile the dependencies in the libavif work tree like we do now and simply install libavif? We could also install the packages for the dependencies.

Reply:
This tests building and installing the AV1 codecs and libavif from source code in a brand new Linux computer. We can delete this test.